build/generate-manifests.sh

```diff
@@ -41,37 +41,57 @@ function checkDependencies() {
     echo "Please install yq package, e.g. 'pip install --user yq'"
     exit 1
   fi
-}
 
-function generateServiceMeshControlPlanesCrd() {
-  yq -s -y --indentless '.[] | select(.kind=="CustomResourceDefinition" and .metadata.name=="servicemeshcontrolplanes.maistra.io") | .' ${DEPLOYMENT_FILE} > ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml
+  if ! [ -x "$(command -v operator-sdk)" ]; then
+    echo "Please install operator-sdk package, e.g. 'brew install operator-sdk' or follow directions at https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md"
+    exit 1
+  fi
 }
 
-function generateServiceMeshMemberRollsCrd() {
-  yq -s -y --indentless '.[] | select(.kind=="CustomResourceDefinition" and .metadata.name=="servicemeshmemberrolls.maistra.io") | .' ${DEPLOYMENT_FILE} > ${BUNDLE_DIR}/servicemeshmemberrolls.crd.yaml
+function generateCRDs() {
+  operator-sdk generate crds --crd-version v1
+  mv deploy/crds/maistra.io_servicemeshcontrolplanes_crd.yaml ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml
+  mv deploy/crds/maistra.io_servicemeshmemberrolls_crd.yaml ${BUNDLE_DIR}/servicemeshmemberrolls.crd.yaml
+  mv deploy/crds/maistra.io_servicemeshmembers_crd.yaml ${BUNDLE_DIR}/servicemeshmembers.crd.yaml
+  rm -rf deploy/crds/
+
+  cat ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml >deploy/src/crd.yaml
```

Comment:
Is there a better way to do this?  `oc apply -f` can work off a directory, so is it necessary to merge all this into a single file?  (Maybe that's required to support the manifest generation, if so, this is an incremental step toward a more conventional operator-sdk structured project?)

Reply:
Yeah, I wanted to keep the existing files for now. If we move to the `apply -f dir/` solution, we'll need to split the `src/` dir into two - `maistra` and `servicemesh`.

Reply:
Sounds good.